todoman/model.py

```diff
@@ -585,8 +586,13 @@ def todos(self, all=False, lists=[], urgent=False, location='',
             q = ', '.join(['?'] * len(lists))
             extra_where.append('AND files.list_name IN ({})'.format(q))
             params.extend(lists)
-        if urgent:
-            extra_where.append('AND priority = 9')
+        if priority is not None:
+            if priority == 0:
+                extra_where.append('AND PRIORITY <= 9 AND PRIORITY >= ?')
+                params.append('{}'.format(priority))
+            else:
+                extra_where.append('AND PRIORITY > 0 AND PRIORITY <= ?')
+                params.append('{}'.format(priority))
```

Comment:
The entire code can be collapsed to:

    if priority:
        extra_where.append('AND PRIORITY > 0 AND PRIORITY <= ?')
        params.append('{}'.format(priority))

Because in the case of `priority == 0` basically no filtering is done, so we don't need to append anything to `extra_where`.